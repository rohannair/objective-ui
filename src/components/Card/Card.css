.card {
  background-color: #fff;
  padding: 2%;
}
