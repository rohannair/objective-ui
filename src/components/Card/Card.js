import React from 'react';
import styles from './Card.css';

import classNames from 'classnames/bind';

let cx = classNames.bind(styles);

export default c => {
  const classname = cx({
    [styles.card]: true,
    [styles.mb]: c.mb
  });

  return (
    <div className={classname}>{c.children}</div>
  );
};
