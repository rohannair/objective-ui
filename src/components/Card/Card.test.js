import React from 'react'
import Card from './index.js'

describe('Test suite for Card component', () => {
  it('Card should exist', () => {
    let wrapper = shallow(<Card/>)
    expect(wrapper).toMatchSnapshot()
  })
})

