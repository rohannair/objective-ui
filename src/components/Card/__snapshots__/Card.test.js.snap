exports[`Test suite for Card component Card should exist 1`] = `
<div
  className="" />
`;
